src/action/base/create_directory.rs

```diff
@@ -5,18 +5,23 @@ use nix::unistd::{chown, Group, User};
 
 use tokio::fs::{create_dir, remove_dir_all};
 
+use crate::action::StatefulAction;
 use crate::{
     action::{Action, ActionDescription, ActionState},
     BoxableError,
 };
 
+/** Create a directory at the given location, optionally with an owning user, group, and mode.
```

Comment:
Is there a reason you use `/* */` comments instead of `//`? Doesn't affect anything, but I almost never see these :laughing:

Reply:
Yes! I prefer multi-line doc comments to make adding examples easier later. (Also I always forget the syntax, so I just lean into using it always instead)